### ISSUE TITLE
Web Accessibility: Less Stick, More Carrot

### ISSUE THREAD

**Issue by jonathana** [dcjs/talks#52]
Your Name: Jonathan Altman
Your twitter handle: @async_io
A few words about yourself: Whole-system engineer, software engineer who isn't good at generating graphically pleasing front-ends from scratch

Talk title: Web Accessibility: Less Stick, More Carrot

Talk abstract:  
Typically, 508 compliance is done by hiring consultants to either explain everything we did wrong with our front end or to either implement the accessibility or sit with the developers to tell them how as they do it.  Wouldn’t it be better if we had our toolkits make it easier to be better on accessibility compliance.  Remember Twitter Bootstrap?  Until it came along, while a developer could make a decently laid-out, well-designed front end without being a graphic designer, it was harder.  Shouldn’t we be making our React/Vue/Angular/Web Components components similarly easy to use by incorporating accessibility in them?

Talk will cover what’s wrong about accessibility process today, what developer experience we should be providing to developers building front-ends, and an exploration of how to start making accessibility easy in at least one front-end toolkit (probably React+Semantic-UI or maybe React-bootstrap).

Expected length: 30-45 minutes

Available months: July or later

- [ ] DCJS[59] Thu Jun 14 2018 
- [X] DCJS[60] Thu Jul 12 2018
- [X] DCJS[61] Thu Aug 09 2018
- [X] DCJS[62] Thu Sep 13 2018
- [X] DCJS[63] Thu Oct 11 2018
- [X] DCJS[64] Thu Nov 08 2018
- [X] DCJS[65] Thu Dec 13 2018
- [ ] DCJS[66] Thu Jan 10 2019
- [ ] DCJS[67] Thu Feb 14 2019
- [ ] DCJS[68] Thu Mar 14 2019
- [ ] DCJS[69] Thu Apr 11 2019
- [ ] DCJS[70] Thu May 09 2019
- [ ] DCJS[71] Thu Jun 13 2019

**Comment by jakerella** [comment]
Hi Jonathan! We had an accessibility talk not too long ago, so we might hold off on this one for a bit so people aren't seeing too much of the same topic. That said, this is important and I think we should work it into the schedule again!

**Comment by jakerella** [comment]
Hi Jonathan! Sorry it's taken so long, but we're hoping you'd be interested in giving this talk in February or March?

**Comment by jonathana** [comment]
What would the date for either of those be?

**Comment by jordangov** [comment]
Feb 14 and Mar 14

**Comment by jakerella** [comment]
Hi! It would be February 14 or March 14... or we can do later, let me know!

**Comment by jakerella** [comment]
Hey @jonathana! Never heard back, but maybe you're up for doing this talk June 13 or July 11?

**Comment by lizafloyd** [comment]
Hi @jonathana ! Just wanted to reach back out and see if you were still interested in giving this talk? If so would love to get you on the schedule. Thanks so much!
